kotsadm/pkg/handlers/ping.go

```diff
@@ -15,6 +15,7 @@ type PingResponse struct {
 	SnapshotInProgressApps []string `json:"snapshotInProgressApps"`
 }
 
+// NOTE: there is no auth on this route
```

Comment:
here

Reply:
ping should work when user is not logged in.  otherwise there will be the "disconnected" modal.  it returns no sensitive data.